server/player/player.go

```diff
@@ -2617,8 +2646,11 @@ func (p *Player) OnGround() bool {
 
 // EyeHeight returns the eye height of the player: 1.62, or 0.52 if the player is swimming.
```

Comment:
Would be nice if you could update this doc as previously mentioned to describe all states